src/main/resources/application.properties

```diff
@@ -33,6 +33,10 @@ paymentTransactionsGateway.uri=${PAYMENT_TRANSACTIONS_GATEWAY_URI}/
 paymentTransactionsGateway.readTimeout=${PAYMENT_TRANSACTIONS_GATEWAY_READ_TIMEOUT}
 paymentTransactionsGateway.connectionTimeout=${PAYMENT_TRANSACTIONS_GATEWAY_CONNECTION_TIMEOUT}
 paymentTransactionsGateway.apiKey=${PAYMENT_TRANSACTIONS_GATEWAY_API_KEY}
+npg.uri=${NPG_URI}/
```

Comment:
why this trailing `/`

Reply:
It's a typo. To be sure that it works, i'll redeploy on dev env after fix

Reply:
It works without "/"